test/lib/ufe/testAttribute.py

```diff
@@ -217,13 +220,17 @@ def testAttributeBool(self):
         # Compare the initial UFE value to that directly from USD.
         self.assertEqual(ufeAttr.get(), usdAttr.Get())
 
-        # Set the attribute in UFE with the opposite boolean value.
-        ufeAttr.set(not ufeAttr.get())
+        # check to see if the attribute edit is allowed
+        isAttrEditAllowed = mayaUsdUfe.isAttributeEditAllowed(usdAttr)
+        self.assertFalse(isAttrEditAllowed, False)
```

Comment:
This is weird...  assertFalse() takes a single argument...  And if you're expecting no edits allowed, then line 226 is a bit pointless :)  Same in a bunch of places.

Reply:
@ppt-adsk  Good catch. Copy-paste damage  :) 

Addressed in 7be71c51668f70493e0533da23f91b1368e73bad

I agree that the check after is useless by my thinking was the code was still valuable to have around. I can remove them if it makes things more clear.

Reply:
As mentioned in the code, I'm concerned that we've lost code coverage because of this.  Would be nice to rework the test file perhaps, so that we can keep the tests?